azurerm_monitor_action_group/main.tf

```diff
@@ -0,0 +1,17 @@
+# The module configures basic monitoring configuration
+
+data "azurerm_resource_group" "rg" {
+  name = "${local.azurerm_resource_group_name}"
+}
+
+
+resource "azurerm_monitor_action_group" "monitor_action_group" {
+  name                = "${var.monitoring_ag_name}"
```

Comment:
by convention, this should be "azurerm_monitor_action_group_name". The same applies to all variables, locals and outputs below